lib/layouts/SubHeader.js

```diff
@@ -162,6 +162,7 @@ const SubHeader = ({
               onFocus={() => handleSearchVisibilty(true)}
               nakedInput
               {...searchProps}
+              clear={null}
```

Comment:
@vyshnavrajs2019 _a
Move the line above `{...searchProps}` so that we override if required. Why is clear required?

Reply:
Clear is not required in the Input component. But when we pass clear prop in searchProps it is also being passed into Input component, hence we get warning message as this is an invalid prop. @edwinbbu _a

Reply:
@vyshnavrajs2019 _a Okay. Can you `clear={null}` above `{...searchProps}`

Reply:
@edwinbbu If I move `clear={null}` above `{...searchProps}` then the prop will be passed to Input. Here I am trying to not pass the clear attribute into Input. Actually Input is receiving a clear prop whose value is null but this is not creating any console warning. Otherwise it will produce a console warning if there is a clear prop (function) in searchProps.